travis/build-and-test.sh

```diff
@@ -44,9 +44,14 @@ function downstream_build_and_test {
     # The build commands cmake, make and ctest must be given as
     # separate commands and not chained with &&. If chaining with &&
     # is used the 'set -e' does not exit on first error.
-    cmake ../ -DENABLE_PYTHON=ON -DBUILD_TESTING=ON -DSILENCE_EXTERNAL_WARNINGS=True -DUSE_QUADMATH=OFF -DADD_DISABLED_CTESTS=OFF
-    make
-    ctest --output-on-failure
+    if [ "$project" == "ewoms" ]; then
+       cmake ../ -DENABLE_PYTHON=ON -DSILENCE_EXTERNAL_WARNINGS=ON -DUSE_QUADMATH=OFF -DADD_DISABLED_CTESTS=OFF
```

Comment:
I guess from the code here that -DBUILD_TESTING defaults to OFF for ewoms, but it would be clearer to explicitly set it here (as in the other branch).